### PR TITLE
convert scale in blas to dnn

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

pass scales in blas layer to dnn layer when converting

## How was this patch tested?

unit tests

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/ReflectionUtils.scala [hunk 1]
```diff
@@ -104,9 +109,23 @@ private[bigdl] object ReflectionUtils {
...
     val element = IRElement[T](
       layer.getName(), op, weights = weightsAndBias._1, gradWeights = weightsAndBias._2)
+    if (layer.isInstanceOf[MklInt8Convertible]) {
+      scales(layer.asInstanceOf[MklInt8Convertible], element)
+    }
     element
   }
 
+  // put scales in fromEle to toELe
+  private def scales[T: ClassTag](fromEle: MklInt8Convertible, toELe: MklInt8Convertible): Unit = {
```
Comment (wzhongyuan) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/ReflectionUtils.scala]: change to `setScales` ?

Comment (zhangxiaoli73) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/ReflectionUtils.scala]: good idea, changed

**Comment by wzhongyuan** [comment]
Please merge it after a PASS run

**Comment by wzhongyuan** [comment]
Jenkins passed  http://mlt-bddl01.sh.intel.com:8080/job/BigDL-PRVN-lite/447/

**Pull request merged**